molecule/default/molecule.yml

```diff
@@ -18,6 +18,8 @@ platforms:
       - /tmp
       - /run
       - /run/lock
+    published_ports:
```

Comment:
I would rather keep `published_ports` at least commented, It is better for CI systems.